odoo/addons/base/models/ir_embedded_actions.py

```diff
@@ -75,7 +75,8 @@ def _compute_is_visible(self):
             active_model_record = self.env[parent_res_model].search(domain_id, order='id')
```

Comment:
For master:
I feel this compute method is a little bit hard to to read, could you check if we could refactor it a little for master?
1. `active_model_record = self.env[parent_res_model].search(domain_id, order='id')`
can we use `active_model` in the context and only set `True` to `record.parent_res_model == active_model`?
2. `record.parent_res_id in (False, self.env.context.get('active_id', False))`
can be `record.parent_res_id in (False, active_id)`